classes/ShoppingfeedApi.php

```diff
@@ -295,29 +295,17 @@ public function getTicketsByReference($taskOrders)
         return $tickets;
     }
 
-    public function getUnacknowledgedOrders($iShipped = false)
```

Comment:
* Please do not change name of a public function in a minor version.
* Do not change arguments name or order. You can only add new arguments and be careful to have an same behaviour with and without the new argument by adding a default value on the args.